src/lib/files.ts

```diff
@@ -40,8 +46,7 @@ function readFile(filePath: string): string {
  */
 function directoryExists(filePath: string): boolean {
   try {
-    return fs.statSync(filePath)
-      .isDirectory();
+    return fs.statSync(filePath).isDirectory();
```

Comment:
I'd recommend not using return statements inside of a try...catch block. There are some interesting behaviors involving try blocks, which can make your code harder to read.
For example:
```
try {
   return 0
} finally {
    cleanup();
}
```
in the above statement, the finally block will still execute after the return statement has run.

Reply:
Correct. Changes like these will be implemented in our refactoring ticket to come. These are only showing now because when I pulled the latest updates the vscode project settings had lint on save enabled.